public/data/cpp.json

```diff
@@ -16,5 +16,90 @@
         "author": "Vaibhav-kesarwani"
       }
     ]
+  },
+  {
+    "categoryName": "Array Manipulation",
+    "snippets": [
+      {
+        "title": "Find Minimum and Maximum Values in an Array",
+        "description": "Finds the minimum and maximum values in an array using only a single loop.",
+        "code": [
+          "void findMinMax(const int arr[], int size, int& min, int& max) {",
+          "    min = arr[0];",
+          "    max = arr[0];",
+          "    for (int i = 1; i < size; i++) {",
+          "        if (arr[i] < min) {",
+          "            min = arr[i];",
+          "        } else if (arr[i] > max) {",
+          "            max = arr[i];",
+          "        }",
+          "    }",
+          "}"
+        ],
+        "tags": ["cpp", "array", "loop", "min-max", "min", "max"],
+        "author": "shovan04"
+      },
+      {
+        "title": "Find Second Largest Element in an Array",
```

Comment:
Also instead of this function, can you make a generalised function that returns Largest `k` elements?